_includes/auth/jwt-provider-endpoint-url.md

```diff
@@ -0,0 +1,7 @@
+## JWT provider endpoint URL
+
+The easiest way to setup JWT authentication against {{site.cloudname}} services is to create a JWT provider endpoint. This endpoint takes a JSON HTTP POST request and produces a JSON result with the token that the service will then use for all the HTTP requests.
```

Comment:
```suggestion
The easiest way to setup JWT authentication for {{site.cloudname}} services is to create a JWT provider endpoint. This endpoint takes a JSON HTTP POST request and produces a JSON result with the token that the service can then use for all HTTP requests.
```
maybe? @lnewson please check?

Reply:
`JSON response` maybe instead of `result`? LGTM otherwise though.

Reply:
updated my suggestion